simple-weather/simple-weather/Controller/ViewController.swift

```diff
@@ -49,7 +56,7 @@ class ViewController: UIViewController, CLLocationManagerDelegate {
     func locationManager(_ manager: CLLocationManager, didUpdateLocations locations: [CLLocation]) {
         guard let locValue: CLLocationCoordinate2D = manager.location?.coordinate else { return }
         print("locations = \(locValue.latitude) \(locValue.longitude)")
-        locationLabel.text = "\(locValue.latitude) \(locValue.longitude)"
+        locationLabel.text = "location\(locValue.latitude) \(locValue.longitude)"
```

Comment:
Still, we are not displaying coordinates as location, we use coordinates to make API requests.